scripts/release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+safe() {
+	$@
+	if [ $? -ne 0 ]; then
+		echo "Build error"
+		exit 1
+	fi
```

Comment:
The purpose of this script is to provide a function that can be called with a command of arbitrary length and arguments. The function runs the command and if it fails, it aborts the script it was invoked from to prevent any additional build commands from being called.

Do you still think it needs changing?